sensors/current_sensor.h

```diff
@@ -0,0 +1,15 @@
+#ifndef CURRENT_SENSOR_H
+#define CURRENT_SENSOR_H
+
+class Current_Sensor : public Sensor
+{
+public:
+    Current_Sensor(const int &pin);
+
+    virtual void read_sensor_value();
```

Comment:
Not 100% sure on this, but I believe that we don't need the virtual keyword here.

Reply:
Yep u totally right